tools/circleci/.gitignore

```diff
@@ -0,0 +1 @@
+images/
```

Comment:
The `.gitignore` format is a tricky one, and this isn't quite right. Take a look at the block comment (which I wrote) at the top of our root `.gitignore`.